src/components/AppHeader.jsx

```diff
@@ -400,19 +401,19 @@ class AppHeader extends Component {
         const { match } = this.props;
         if (togglePermalinkUUID) {
             let state = !this.state.permalinkUUID;
-            this.state.permalinkUUID = state;
+            this.state.setState({
+                permalinkUUID: state
+            });
         }
         let j = JSON.stringify(
             BackendAPI.getExportObject(match.params),
             BackendAPI.getExportKeys()
         );
         let d = pako.deflate(j, { to: 'string' });
         let b = encodeURIComponent(window.btoa(d).replace(/\//g, '$'));
-        if (this.state.permalinkUUID) {
-            var uuid = 'permalink__' + match.params.uuid;
-        } else {
-            var uuid = 'permalink';
-        }
+        const uuid =
+            'permalink' + (this.state.permalinkUUID ? match.params.uuid : '');
```

Comment:
This will break permalinks as is. 

```suggestion
            'permalink' + (this.state.permalinkUUID ? "__" + match.params.uuid : '');
```

Reply:
Good catch! Thanks.